README.md

```diff
@@ -3,38 +3,31 @@
 ## Zu installierende Node.js-Pakete
 npm install dotenv
 npm install otpauth
-npm install jimp
-npm install qrcode-reader
 
 # Oft verwendetet Konsolenbefehle
 
 ## Einen bestimmten Test ausführen: 
-npx playwright test TF0001.spec.ts
+USER="xxx" PW="xxx" npx playwright test TF0001.spec.ts
```

Comment:
Woher weiß ich, welche User/PW Kombi ich angeben muss?

Reply:
Aktuell gibt es nur einen Benutzer in keycloak. In Zukunft erstellt playwright die Testdaten über die API.
Bis dahin habe ich diese einfache, flexible Variante gewählt damit man die Tests lokal laufen lassen kann. Wenn sich der Benutzer/PW ändert in keycloak, kann man das lokal ändern und muss nicht das secret in github ändern.
Wie gesagt, dieses ist erstmal der workaround.

Reply:
Sollen wir dann in der Readme noch erwähnen, dass Playwright immer auf test.dev geht, auch wenn man es lokal ausführt?